.textlintrc.json

```diff
@@ -15,7 +15,10 @@
         "/<http://.*>/",
         "medium.com/opentelemetry",
         "github.com/open-telemetry",
-        "#opentelemetry"
+        "#opentelemetry",
+        "/^(?:language|repo): .*$/m",
+        "/tags:([\\n\\r\\s]*-[\\s\\S]*?[\\n\\r]+)*/m",
+        "Instrumentation for net/http"
```

Comment:
I'd vote to add code ticks to:

> title: splunkhttp -- Instrumentation for net/http

so that it reads

> title: splunkhttp -- Instrumentation for `net/http`

instead of adding this one-off rule.

I can address title markdown at the same time as I tackle the markdown in descriptions. WDYT?

Reply:
done